rewrite-test/src/main/kotlin/org/openrewrite/java/JavaVisitorCompatibilityKit.kt

```diff
@@ -16,6 +16,7 @@
 package org.openrewrite.java
 
 import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
```

Comment:
looks like an accidental import and can be removed

Reply:
Go for it